app/server/src/services/time.js

```diff
@@ -0,0 +1,33 @@
+
+function getTimeDifference(startDate, endDate) {
+    // Calculate the difference in milliseconds between the two dates
+    var differenceInMilliseconds = endDate - startDate;
+    
+    // Convert the difference in milliseconds to seconds
+    var differenceInSeconds = differenceInMilliseconds / 1000;
+    
+    // Calculate the difference in minutes
+    var differenceInMinutes = differenceInSeconds / 60;
+    
+    // Calculate the difference in hours
+    var differenceInHours = differenceInMinutes / 60;
+    
+    // Calculate the difference in days
+    var differenceInDays = differenceInHours / 24;
+
+    // Return the time difference as a string
+    if(differenceInDays > 7){
+        return Math.floor(differenceInDays) + " weeks ago";
+    }else if(differenceInDays > 1){
+      return Math.floor(differenceInDays) + " days ago";
+    }else if(differenceInHours > 1){
+      return Math.floor(differenceInHours) + " hours ago";
```

Comment:
This would result in "1 hours ago" in case the difference is between 1 and 2, no?

Reply:
same with the others

Reply:
Yes, what is the issue?

Reply:
We would want to see `1 hour ago` I think.

Reply:
Okay added.